src/measure_theory/set_integral.lean

```diff
@@ -200,6 +267,42 @@ lemma integral_on_congr_of_set (hsm : measurable_on s f) (htm : measurable_on t
   (h : ∀ₘ a, a ∈ s ↔ a ∈ t) : (∫ a in s, f a) = (∫ a in t, f a) :=
 integral_congr_ae hsm htm $ indicator_congr_of_set h
 
+lemma integral_on_empty (f : α → β) : (∫ a in ∅, f a) = 0 :=
+by { rw [indicator_empty, integral_zero] }
+
+lemma integral_on_const (hs : is_measurable s) (b : β) :
+  (∫ a in s, b) = ennreal.to_real (volume s) • b :=
+begin
+  by_cases b0 : b = 0,
+  { simp [b0] },
+  have : (indicator s (λa:α, b)) = restrict (const α b) s := (const α b).indicator_eq_restrict hs,
+  rw this,
+  by_cases h : volume s < ⊤,
+  { rw [integral_eq_bintegral, restrict_const_bintegral _ hs],
+    exact (const α b).fin_vol_supp_restrict hs h },
+  { have : volume s = ⊤, { rwa [ennreal.lt_top_iff_ne_top, not_not] at h },
+    rw [this, ennreal.top_to_real, zero_smul, integral_non_integrable],
+    rwa [integrable_iff_fin_vol_supp, fin_vol_supp_restrict_const hs b0] }
+end
+
+lemma integral_on_volume_zero (hs : volume s = 0) : (∫ a in s, f a) = 0 :=
+begin
+  by_cases h : measurable_on s f,
+  { rw ← integral_zero α β,
+    apply integral_congr_ae h measurable_const,
+    exact indicator_eq_zero_ae hs },
+  rw [integral_on_non_measurable h]
+end
+
+lemma integral_on_singleton (f : ℝ → β) (a : ℝ) : (∫ x in {a}, f x) = 0 :=
+integral_on_volume_zero real.volume_singleton
+
+lemma integral_on_Icc_self (f : ℝ → β) (a : ℝ) : (∫ x in Icc a a, f x) = 0 :=
+by rw [Icc_self, integral_on_singleton]
+
+lemma integral_on_interval_self (f : ℝ → β) (a : ℝ) : (∫ x in [a, a], f x) = 0 :=
+by { rw [interval_self, integral_on_singleton] }
```

Comment:
These lemmas are specific to Lebesgue measure on the real line, while all the statements around are general. Would it make sense to move them elsewhere (in a different section of this file, or a different file concerned specifically with integrals on the real line)?